tensilelite/Tensile/KernelWriter.py

```diff
@@ -3449,6 +3451,19 @@ def readWriteVectors(mat, vw, kernel):
     # for conditionals
     self.states.lastPostLoopSgpr = self.sgprPool.size()
 
+    self.states.preloadScaleA = False
```

Comment:
do we need this ?
If so, it also need to init preloadScaleB

Reply:
fixed